feat: docs for multi-container metrics gathering configurator (#1467)
mia-platform/documentation: docs/development_suite/api-console/api-design/microservice-monitoring.md

title: Microservice Monitoring
sidebar_label: Monitor Service metrics
---

To foster application observability you can set up metrics gathering configuration for your desired monitoring system.

:::tip
Find out more about supported monitoring systems and how to [configure them in the Project Settings](/console/project-configuration/project-settings.md#runtime)!
:::

:::info

> If no Monitoring system is configured for your Project you will not be able to set up any metric, head to the [the Project Settings](/console/project-configuration/project-settings.md#runtime)

Reviewer: ```suggestion
If no Monitoring system is configured for your Project you will not be able to set up any metric: head to the [the Project Settings](/console/project-configuration/project-settings.md#runtime)
```

or ask your Project Administrator in order to activate the feature!
:::

## Configure metrics gathering

> Once the Runtim Monitoring system is configured for your Project, you will be able to define scraping rules for each of your microservices container.

Reviewer: ```suggestion
Once the Runtime Monitoring system is configured for your Project, you will be able to define scraping rules for each of your microservices container.
```

> In the Design section, each microservices features the Monitoring tab where you can view, udpate or delete your metrics scraping endpoint.

Author: ```suggestion
In the Design section, each microservices features the Monitoring tab where you can view, update or delete your metrics scraping endpoint.
```

> In the Design section, each microservices features the Monitoring tab where you can view, update or delete your metrics scraping endpoint.

Reviewer 2: ```suggestion
In the Design section, each microservice features the Monitoring tab where you can view, update, or delete your metrics scraping endpoint.
```

> For each container you can specificy multiple metrics endpoint, defined by the following fields

Reviewer: ```suggestion
For each container you can specificy multiple metrics endpoint, defined by the following fields:
```

> For each container you can specificy multiple metrics endpoint, defined by the following fields:

Reviewer 2: ```suggestion
For each container you can specify multiple metrics endpoints, defined by the following fields:
```
